src/NOTD.SC2Map/enUS.SC2Data/LocalizedData/GameStrings.txt

```diff
@@ -1606,7 +1608,7 @@ Button/Tooltip/Inception=The universe around you slows down. You are free to act
 Button/Tooltip/IncompleteIvax=Heavy assault mech. Can use 250mm Strike Cannons.<n/><n/><c val="#ColorAttackInfo">Can attack ground and air units.</c>
 Button/Tooltip/InfantrySpecializationLevel1=Allows Mobile Infantry to land Flamethrowers. You can customize your drop team in a sub-menu.
 Button/Tooltip/InfantrySpecializationLevel2=Allows Mobile Infantry to land Snipers. You can customize your drop team in a sub-menu.
-Button/Tooltip/Inferno=Fires incendiary pallets in a forward arc that burns into the first 6 units it encounters (12 max range). Units with incendiary pallets will begin burning from within and explode after 5 seconds, dealing 200 damage to all ground units in 3 radius.
+Button/Tooltip/Inferno=Fires incendiary pallets in a forward arc that burns into the first 6 units it encounters (12 max range). Units with incendiary pallets are slowed by 60% while burning from within and explode after 5 seconds, dealing 200 damage to all ground units in 3 radius. Non-heroic armored biological ground units that die to the blast burst into pillars of fire that last 10 seconds. Units caught in the flames have a 40% chance per second of spreading Inferno, potentially causing a chain reaction.<n/><n/><c val="FF0000">Warning: Inferno creates fires that can engulf friendlies. Use with extreme caution!</c><n/><n/>"When the Saracens came to attack him, they threw Greek fire onto the barrier... the Turks did not wait for the fire to burn itself out, but rushed upon the Templars among the scorching flames." - Jean de Joinville

```

Comment:
^ real quote I found about "Greek Fire" (I thought it would suit the chain reaction nature of the new inferno.)